client/app/components/conference_room/provider/EventProvider.jsx

```diff
@@ -77,6 +78,17 @@ export default class EventProvider extends React.Component {
       .catch(() => this._toggleConfirmed());
   }
 
+  _finishEvent() {
+    if (typeof(this.state.currentEvent) === 'undefined') {
+      return;
+    }
+
+    const currentEvent = this.state.currentEvent;
+    this.setState({ currentEvent: undefined });
+    EventSource.finish(this.props.conferenceRoom.id, this.state.currentEvent.id)
```

Comment:
I believe it should be `EventSource.finish(this.props.conferenceRoom.id, currentEvent.id)`